src/lib/LLVMExec.hs

```diff
@@ -358,6 +361,7 @@ compileCUDAKernel logger (LLVMKernel ast) = do
         usePTXAS <- maybe False (=="1") <$> lookupEnv "DEX_USE_PTXAS"
         if usePTXAS
           then do
+            ptxasPath <- (flip (++) $ "/bin/ptxas") <$> cudaPath
```

Comment:
wouldn't `(++"/bin/ptxas")` work?

Reply:
Ah cool, didn't know you could use concat like that.

Reply:
Yeah you can do that with any binary operator. `(+2)` and `(2*)` are also valid!